src/kep/validation/checkpoints.py

```diff
@@ -186,12 +186,12 @@ def as_dict(s):
 
 
 def extract_date(d):
-    year = int(d.pop('year'))
+    year = int(d.get('year'))
```

Comment:
I had to change `pop` to `get` here to avoid false positives because dataframes contains these columns (`year`, `month`, `qtr`) too.

Reply:
Can you add this comment to code? It is important clarification